idl/github.com/uber/tchannel/meta.thrift

```diff
@@ -22,7 +39,7 @@ struct VersionInfo {
 }
 
 service Meta {
-    HealthStatus health()
+    HealthStatus health(1: HealthRequest hr)
```

Comment:
Yeah, this change is backwards compatible over-the-wire and with existing services which is really all that matters. The only breakage will be that your old code will fail to compile until you address the new parameter. This is completely acceptable.